Doc/c-api/call.rst

```diff
@@ -235,14 +246,16 @@ please see individual documentation for details.
    .. versionadded:: 3.9
 
 
-.. c:function:: PyObject* _PyObject_CallOneArg(PyObject *callable, PyObject *arg)
+.. c:function:: PyObject* PyObject_CallOneArg(PyObject *callable, PyObject *arg)
 
    Call a callable Python object *callable* with exactly 1 positional argument
    *arg* and no keyword arguments.
 
    Return the result of the call on success, or raise an exception and return
    *NULL* on failure.
 
+   This function is not part of the `limited API <stable>`_.
```

Comment:
I would prefer to add it to the limited API, see previous comments for the rationale.

Reply:
Oh wait. I didn't notice that _PyObject_CallOneArg() is currently a static inline function. Maybe static inline functions should not be exposed in the limited API? I opened a thread on python-dev: https://mail.python.org/archives/list/python-dev@python.org/thread/P7RURGS4ZG4RVKEVFFP36BP6BUU6CBX5/

Reply:
Putting it in the stable API would mean writing it as a regular function.